### PR TITLE
Ammo.xml repath

### PR DESCRIPTION
**Pull request by ghost**
this is not a place of honor. there is no glory here. this is a dark, terrible place of our creation.

### CODE REVIEW

File: Items/Weapons/Ammo/smg_ammo.xml [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+        <SpawnItem identifier="scp_valapprojectile" spawnposition="ThisInventory" />
+        <ParticleEmitter particle="scp_spentcasingriflesmall" emitinterval="1.5" particleamount="1" particlespersecond="50" anglemin="70" anglemax="100" velocitymin="250" velocitymax="300" copyentityangle="true" scalemin="0.35" scalemax="0.35" />
+      </StatusEffect>
+
+    </ItemContainer>
+    
+    <Holdable canBeCombined="true" slots="Any,RightHand,LeftHand" handle1="0,0" msg="ItemMsgPickUpSelect" />
+
+  </Item>
+
```
Comment (ven1883) [inline Items/Weapons/Ammo/smg_ammo.xml:41]: missing separation indicators, use !--###-- comments

File: Items/Weapons/Ammo/smg_ammo.xml [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+      <StatusEffect type="OnImpact" target="UseTarget">
+        <Conditional entitytype="eq Structure" />
+        <Conditional hastag="eq door" />
+      </StatusEffect>
+      
+    </Projectile>
+
+  </Item>
+
```
Comment (ven1883) [inline Items/Weapons/Ammo/smg_ammo.xml:92]: some items do not have names attached to them, do this!!

**Review by ven1883 — CHANGES_REQUESTED** [review]
also you need to clean out and remove the ammo.xml file itself otherwise I have to do it manually

**Comment by WrillWasTaken** [comment]
> also you need to clean out and remove the ammo.xml file itself otherwise I have to do it manually

image

File: Items/Weapons/Ammo/smg_ammo.xml (hunk 1 above)
Comment (WrillWasTaken) [reply Items/Weapons/Ammo/smg_ammo.xml:41]: done

**Comment by ven1883** [comment]
i gaveth you a PR to work with

also the fixme file was NOT deleted!!!!!!!!! it is in my PR

**Comment by ven1883** [comment]
before i go insane wrill asked me to finish this

**Review by ven1883 — COMMENTED** [review]
Review system is goofy

**Pull request merged**